### PR TITLE
Add option for preventing default link action.

### PR DESCRIPTION
**Pull request by ajdougal**
This will prevent the page from resetting to the top when the jQueryFileTree is very large.

### CODE REVIEW

**Comment by daverogers** [comment]
Do you have an example where it's resetting to the top?

**Comment by ajdougal** [comment]
I can give you one tomorrow, I'll have to pull up an old branch of ours. We
went ahead and are using my new change off a local branch right now.

It's mainly because file tree uses links as the folder clicking mechanism,
and the default behavior in HTML for clicking a link is to reload the page
and move to the top. In order to see the bug, your browser window has to be
in a scrolled state when you click the link

**Comment by vinceis1337** [comment]
@daverogers I work with @ajdougal here is are some example gifs:

[jqueryfiletree before](https://giphy.com/gifs/E1jPsivhFj8WY)
[jqueryfiletree after](https://giphy.com/gifs/3o6gEaiUhdEVkNngUo)

**Comment by daverogers** [comment]
Ah. That's interesting.

Change seems fine, but I need you to update the Coffeescript file (jqueryfiletree/src/coffeescript/) and then run the default Gulp task. I realize this is confusing at first glance, and something I need to change sometime in the future because you're not the first to modify that file directly. The Gulp file will generate the dist .min files as well.

**Comment by vinceis1337** [comment]
@daverogers coffeescript changes implemented and gulped as requested! Thanks!

**Comment by vinceis1337** [comment]
Okay, it looks like the css changes were not put in the .less file and the gulp destroyed them. I'll revert the css changes.

**Comment by vinceis1337** [comment]
@daverogers less changes have been implemented and now preserve the changes introduced in [Pull Request](https://github.com/jqueryfiletree/jqueryfiletree/pull/30) https://github.com/jqueryfiletree/jqueryfiletree/pull/30

**Comment by vinceis1337** [comment]
@daverogers I've added a section to the README.md file that indicates that the changes must be done in the .coffee/.less files or else the changes will not be preserved.

**Comment by daverogers** [comment]
Changes look good 👍  Thanks for all the effort.

**Comment by vinceis1337** [comment]
Ok looks like I accidentally [swept up a comment](https://github.com/jqueryfiletree/jqueryfiletree/pull/41/commits/ecefaf52241993055564cc35093b431c7acd788a#diff-c40c5cffcc2c27a5b8d1f15273bb63a5R15) in my commit (I was trying to properly add comments in the css file). It doesn't really matter though as it will be wiped away after the next gulp.

File: src/jQueryFileTree.css [hunk 1]
```diff
@@ -12,21 +12,8 @@ UL.jqueryFileTree LI {
   padding-left: 20px;
   margin: 0;
   white-space: nowrap;
+  /* File Extensions */
```
Comment (vinceis1337) [inline src/jQueryFileTree.css]: This shouldn't be here, and will disappear after the next gulp.

**Comment by daverogers** [comment]
Ah, yeah, no problem. Was just going to shoot out a minor release anyways, so it'll be gone here in a second.

**Pull request merged**